config/locales/account_reset/en.yml

```diff
@@ -1,6 +1,11 @@
 ---
 en:
   account_reset:
+    cancel_request:
+      are_you_sure: Are you sure you want to cancel your delete account request?
+      cancel: Exit
```

Comment:
Is `Exit` the right word to use? We don't use that word anywhere else in our app. Perhaps `Cancel` or `Back to sign in page`?

Reply:
Eric had an issue with Cancel since the main purpose of the screen is to cancel something.  Back to sign in page doesn't make sense either because they landed on the screen directly from a link in their email.